package.json

```diff
@@ -70,10 +70,23 @@
     "node": ">=16.13.0",
     "npm": ">= 8.1.0"
   },
+  "peerDependencies": {
+    "@seamapi/types": "^1.0.0",
+    "type-fest": "^4.0.0"
+  },
+  "peerDependenciesMeta": {
+    "@seamapi/types": {
+      "optional": true
+    },
+    "type-fest": {
+      "optional": true
+    }
+  },
   "dependencies": {
-    "@seamapi/types": "^1.12.0"
+    "axios": "^1.5.0"
```

Comment:
should this be a peer too?

Reply:
Good question. I think no in this case. The reason for the types peer deps is a pattern that seems to solve the ts / js consumer issue of types are only deps for ts consumers.

Reply:
my original thought prompting this was if consumers wanted to add guards for `AxiosError` but forgot we throw custom errors :)